2089_find_target_indices.py

```diff
@@ -0,0 +1,15 @@
+from typing import List
+
+
+class Solution:
+    def targetIndices(self, nums: List[int], target: int) -> List[int]:
```

Comment:
Insert a new line before method declaration.

Reply:
Sure, I changed it